### DISCUSSION TITLE
Component Proposal: Toggle

### DISCUSSION THREAD

**Discussion by akinhwan** [uswds/uswds#5772] [Proposals]
![image](https://user-images.githubusercontent.com/10124722/69280933-5ae27c80-0bb5-11ea-9e99-7477d1e6457c.png)
I was looking for a toggle component, but unfortunately couldn't find one in the documentation. 
Although I do see it used as an icon on the landing page :)

Was this intentional to leave it out? Seems like it might be a good fit and possible addition to the "Form Controls" section https://designsystem.digital.gov/components/form-controls/

**Comment by iamandrewmiller** [comment]
Any movement on this request?

**Comment by sovanyio** [comment]
Seems that the intended usage is to have a single checkbox: https://designsystem.digital.gov/form-controls/02-checkboxes/#checkbox-docs

**Comment by nektro** [comment]
A toggle is different from a checkbox because the user expectation is that a checkbox is only changing state of the local form and does not contact the server etc. Whereas a toggle is seen as a button and *does* update external state on its own.
